db/migrate/20241021223548_add_peer_tube_url_to_videos.rb

```diff
@@ -0,0 +1,5 @@
+class AddPeerTubeUrlToVideos < ActiveRecord::Migration[7.2]
+  def change
+    add_column :videos, :peer_tube_url, :text
```

Comment:
Isn't text really huge compared to string or they are the same on the database?

Reply:
it doesn't much matter these days. `text` also gives more flexibility for possibly looooooong URLs. which probably isn't the case with a peertube URL, but it's just a pattern that i always follow for URL columns in a (Postgres) database. 🤙🏻